client/client.go

```diff
@@ -17,17 +17,17 @@ import (
 
 const authorizationHeader = "Authorization"
 
-// Client is a low level interface for clients of the elements protocol
+// Client is a low level interface for clients of the elements protocol.
 type Client interface {
 	Request(ctx context.Context, options RequestOptions) (*http.Response, error)
```

Comment:
This should conform to `http.RoundTripper`, but we can change that another time.